com.adsbynimbus.nimbus/VersionConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace Nimbus
+{
+    public static class VersionConstants
+    {
+        public const string UnitySdkVersion = "1.8.1";
```

Comment:
This version should not be hardcoded; using the Package List command from the previous comment, you can search for packageId = `com.adsbynimbus.nimbus` and then pass the PackageInfo.version field into the networking client.